selfhost/typechecker.jakt

```diff
@@ -4175,7 +4193,10 @@ struct Typechecker {
         }
         SingleQuotedString(val, span) => CheckedExpression::CharacterConstant(val, span)
         SingleQuotedByteString(val, span) => CheckedExpression::ByteConstant(val, span)
-        QuotedString(val, span) => CheckedExpression::QuotedString(val, span)
+        QuotedString(val, span) => {
+            .unify_with_type(found_type: TypeId::from_string("0_13"), expected_type: type_hint, rhs_span: span)
```

Comment:
What is the significance of `"0_13"` here? There is no comment, or explanation in the commit message.

Reply:
replaced it for builtin(BuiltinType::String) which represents the same thing. Much more readable now.

Reply:
Great!